src/Microsoft.ML.Transforms/NAIndicatorTransform.cs

```diff
@@ -4,25 +4,36 @@
 
 using System;
 using System.Collections.Generic;
+using System.Linq;
 using System.Text;
+using Microsoft.ML.Core.Data;
 using Microsoft.ML.Runtime;
 using Microsoft.ML.Runtime.CommandLine;
 using Microsoft.ML.Runtime.Data;
 using Microsoft.ML.Runtime.Data.Conversion;
 using Microsoft.ML.Runtime.EntryPoints;
 using Microsoft.ML.Runtime.Internal.Utilities;
 using Microsoft.ML.Runtime.Model;
+using Microsoft.ML.StaticPipe;
+using Microsoft.ML.StaticPipe.Runtime;
+using Microsoft.ML.Transforms;
 
-[assembly: LoadableClass(typeof(NAIndicatorTransform), typeof(NAIndicatorTransform.Arguments), typeof(SignatureDataTransform),
-   NAIndicatorTransform.FriendlyName, "NAIndicatorTransform", "NAIndicator", NAIndicatorTransform.ShortName, DocName = "transform/NAHandle.md")]
+[assembly: LoadableClass(NAIndicatorTransform.Summary, typeof(IDataTransform), typeof(NAIndicatorTransform), typeof(NAIndicatorTransform.Arguments), typeof(SignatureDataTransform),
+    NAIndicatorTransform.FriendlyName, nameof(NAIndicatorTransform), "NAIndicator", NAIndicatorTransform.ShortName, DocName = "transform/NAHandle.md")]
 
-[assembly: LoadableClass(typeof(NAIndicatorTransform), null, typeof(SignatureLoadDataTransform),
-    NAIndicatorTransform.FriendlyName, NAIndicatorTransform.LoaderSignature)]
+[assembly: LoadableClass(NAIndicatorTransform.Summary, typeof(IDataTransform), typeof(NAIndicatorTransform), null, typeof(SignatureLoadDataTransform),
+    NAIndicatorTransform.FriendlyName, nameof(NAIndicatorTransform))]
 
-namespace Microsoft.ML.Runtime.Data
+[assembly: LoadableClass(NAIndicatorTransform.Summary, typeof(NAIndicatorTransform), null, typeof(SignatureLoadModel),
+    NAIndicatorTransform.FriendlyName, nameof(NAIndicatorTransform))]
+
+[assembly: LoadableClass(typeof(IRowMapper), typeof(NAIndicatorTransform), null, typeof(SignatureLoadRowMapper),
+   NAIndicatorTransform.FriendlyName, nameof(NAIndicatorTransform))]
+
+namespace Microsoft.ML.Transforms
```

Comment:
>Transforms [](start = 23, length = 10)

can you fix EntryPointCatalog test?

Reply:
Ok working on it.

---
In reply to: [225648543](https://github.com/dotnet/machinelearning/pull/1217#discussion_r225648543) [](ancestors = 225648543)